.changeset/silly-turkeys-count.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-js': patch
+---
+
+Update setBusySubmitButton to check for submitter button
```

Comment:
How about something like "Add submitter check to ensure only one submit button displays loading state on forms with multiple submit buttons."

Reply:
Much better, thanks!